src/hooks/usePoolData.ts

```diff
@@ -16,6 +16,7 @@ import { UserStateContext } from "./../providers/UserStateProvider"
 import { Zero } from "@ethersproject/constants"
 import { parseUnits } from "@ethersproject/units"
 import { useActiveWeb3React } from "."
+// import { useBasicTokens } from "../providers/TokensProvider"
```

Comment:
Is this comment intentional?

Reply:
Going to remove, this was before the ExpandedPool Provider so is a little outdated.